### DISCUSSION TITLE
Pylon Issue

### DISCUSSION THREAD

**Discussion by ThatBritGamer** [Pryaxis/TShock#1925] [Q&A]
Using your 4.4.0 release 7 build and found that your pylon system isn't working.

I can place the Pylons outside of Tshock and see them on the map and teleport.

If i access that map In tshock i can access that network and teleport, But if i place down a new pylon or move a pylon then it wont seem to work on the network.

I can place the Pylon down and click on it. But whenever i try to teleport to an active pylon it says
"You are not close enough to a pylon to teleport with the pylon network"

And the pylon your on doesn't show on the map as a icon.

Thought i'd mention this.

I haven't tried the Release 8 build yet but from your release notes i dont see anything referencing the Pylon system.

**Comment by hakusaro** [comment]
Sorry -- this is a known issue! We're working on it!
